src/App.css

```diff
@@ -14,7 +14,7 @@
 
 .App__header {
   position: fixed;
-  padding: 30px;
+  padding: 20px;
```

Comment:
In general is better to use `rem` or `em` for paddings.
This little article has some good practices:
https://daveredfern.com/use-units-css/

TLDR: 
- padding is related to the element font-size? -> `em`
- padding is not related to the element font-size -> `rem`

Reply:
thank you!